charts/self-host/templates/admin.yaml

```diff
@@ -79,6 +79,10 @@ spec:
           mountPath: /etc/bitwarden/logs
           subPath: admin
         {{- end }}
+        {{- if .Values.volume.ca_certificates.enabled }}
```

Comment:
I would suggest a generic additional volume mount. Where you could specify additional volume names and paths (in an array or map).
Then certificate mounting could be just one of may use cases.